cylc/flow/task_remote_cmd.py

```diff
@@ -84,17 +85,29 @@ def create_client_keys(srvd, install_target):
     os.umask(old_umask)
 
 
-def remote_init(install_target, rund, indirect_comm=None):
+def remote_init(install_target, rund, *dirs_to_symlink, indirect_comm=None):
     """cylc remote-init
 
     Arguments:
         install_target (str): target to be initialised
         rund (str): suite run directory
+        dirs_to_symlink (list): directories to be symlinked in form
+        [directory=symlink_location, ...]
```

Comment:
Either `*dirs_to_symlink` should be `dirs_to_symlink` in the args, or the docstring should not call it `(list)` (but what do you call unpacked positional args?)

Reply:
I've checked this with @oliver-sanders and unpacked postitional args become a list in docstrings. Does seem a bit non-intuitive though!

Reply:
Yuk, if you pass a list the for loop won't work 😢 Oh well